groups/groups.yaml

```diff
@@ -108,6 +108,10 @@ groups:
       - detiber@gmail.com
       - vince@vincepri.com
       - hi@amycod.es
+  - email-id: k8s-infra-staging-descheduler@kubernetes.io
+    members:
+      - ravisantoshgudimetla@gmail.com
```

Comment:
@dims @spiffxp Do we auto-add folks like yourselves to all groups, or do we expect that to be done here, or do we not care?

Reply:
@spiffxp what do you lean towards? i am ok either way

Reply:
@seanmalloy for now, please add me and Aaron to the list of members for better clarity on who has access

Reply:
Fixed. I added @dims and @spiffxp to the list.